.github/workflows/ci_test.yml

```diff
@@ -25,8 +25,13 @@ jobs:
           python-version: ${{ matrix.python-version }}
       - name: Run python tests
         run: |
-          docker pull twosixarmory/tf1:0.5.0
           pip install -r requirements.txt
+          version=$(python -m armory --version)
+          docker build --file docker/Dockerfile --target armory -t twosixarmory/armory:${version} .
+          docker build --file docker/tf1/Dockerfile --build-arg armory_version=${version} \
+          --target armory-tf1-base -t twosixarmory/tf1-base:${version} .
+          docker build --file docker/tf1-dev/Dockerfile --build-arg armory_version=${version} \
+          --target armory-tf1-dev -t twosixarmory/tf1:${version} .
```

Comment:
Could we make this a script? say `build-dev-minimal.sh` or something?

Reply:
Sounds good.